README.md

```diff
@@ -42,20 +42,19 @@ The `'cookie`' scheme takes the following options:
   query component of the `redirectTo` URI using the parameter name `'next'`. Set to a string to use
   a different parameter name. Defaults to `false`.
 - `redirectOnTry` - if `false` and route authentication mode is `'try'`, authentication errors will
-  not trigger a redirection. Requires **hapi** version 6.2.0 or newer. Defaults to `true`;
-- `validateFunc` - an optional session validation function used to validate the content of the
+  not trigger a redirection. Defaults to `true`;
+- `async validateFunc` - an optional session validation function used to validate the content of the
```

Comment:
Do you think `async` should be here? It could confuse people, thinking that async is part of the property name? In hapi-auth-basic I put it as part of the signature: https://github.com/hapijs/hapi-auth-basic#hapi-auth-basic

Reply:
In hapi docs its in the description signature.